scripts/regenerate-dbsf-oracle.py

```
@@ -0,0 +1,262 @@
+            "description": "Scores cross zero — DBSF still normalizes correctly.",
+            "lists": [
+                [("d-A", 0.7), ("d-B", -0.1), ("d-C", -0.4), ("d-D", -0.7)],
+                [("d-A", 2.1), ("d-B", 0.5), ("d-C", -1.5), ("d-D", -3.0)],
+            ],
+            "top_k": 10,
+            "weights": None,
+        }
+    )
+
+    # 6. Large-skew list — one list has tightly clustered scores around 0,
+    # the other has a wide range. Exercises σ-spread asymmetry.
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `σ` (GREEK SMALL LETTER SIGMA). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF003)